p2pserver/common/p2p_common.go

```diff
@@ -76,6 +76,13 @@ const (
 	ACTOR_TIMEOUT = 5 //actor request timeout in secs
 )
 
+//recent contact const
+const (
+	RECENT_TIMEOUT   = 60
+	RECENT_FILE_NAME = "peers.recent"
```

Comment:
how would it be if node run in mainnet -> restart -> testnet ? may be info should be stored or use different magic number in net message header .

Reply:
In case node would switch between mainnet  and testnet, the magic number should store inmsg header, and i think we should add net magic number in config which user can change it manually.
I will create a new issue about this situation, if check magic number when in hand shake state，node shouldn't`t connect to wrong peers and the recent list will be changed when valid connection setup.